packages/proposals/src/models/initTimelockConfig.ts

```diff
@@ -13,19 +13,19 @@ import * as Layout from '../utils/layout';
 ///           program account key, governance mint key, council mint key, timelock program account key.
 ///   1. `[]` Program account that this config uses
 ///   2. `[]` Governance mint that this config uses
-///   3. `[]` Council mint that this config uses [Optional] [Pass in 0s otherwise]
+///   3. `[]` Council mint that this config uses [Optional]
```

Comment:
I noticed in your other PR (#7) which I merged that you haven't re-arranged Council Mint to be the last argument but here you have. You need to reconcile that.

Reply:
I did rearrange it [#7](https://github.com/bhgames/solana-program-library/pull/7/filesl)